### PR TITLE
Work around hard-coded keyboard bindings on Linux and Windows

### LINKED ISSUE #1027: CTRL+A not working in diagram editors
__Steps to Reproduce__

* Open BPMN diagram
* `CTRL+A` to select all elements
* __Does not work...__

__Tasks__

* [x] Restore [`ShortcutsFix`](https://github.com/camunda/camunda-modeler/blob/master/client/lib/external/shortcuts-fix.js) to support key binding for built-in key strokes on Windows and Linux

### PR DESCRIPTION
**Pull request by nikku**
Closes #1027

### CODE REVIEW

File: client/src/remote/Backend.js [hunk 1]
```diff
@@ -8,8 +8,9 @@ const ids = new Ids();
  */
 export default class Backend {
 
-  constructor(ipcRenderer) {
+  constructor(ipcRenderer, process) {
```
Comment (nikku) [inline client/src/remote/Backend.js]: Is there a reason we don't extract the platform using basic browser OS detection?

**Comment by nikku** [comment]
> Is d770f51 meant to address #1024?

Answer to self: No, issue is still relevant and open with many potential issues (cf. [updated issue](https://github.com/camunda/camunda-modeler/issues/1024)).

**Comment by philippfromme** [comment]
@nikku Ready for review. I've created two seperate issues that I deliberately didn't tackle this time:

1. Integrate CommandStack#undo and CommandStack#redo when working with the properties panel (https://github.com/camunda/camunda-modeler/issues/1059)
2. Decide wether to re-implement custom copy/cut/paste behavior in decision table depending on selection range (https://github.com/camunda/camunda-modeler/issues/1060)

File: client/src/app/KeyboardBindings.js [hunk 2]
```diff
@@ -126,17 +126,7 @@ function isSelectAll(event) {
...
-    .map(s => s.trim().toLowerCase())
-    .join('+');
-
-  b = b
-    .split('+')
-    .map(s => s.trim().toLowerCase())
-    .join('+');
-
-  return a === b;
+  return a.replace(/\s+/g, '') === b.replace(/\s+/g, '');
```
[the remark this answers is not in the document]
Comment (nikku) [reply client/src/app/KeyboardBindings.js]: Why not? What is the case where we'd like to compare for case-insensivity?

**Pull request merged**